### PR TITLE
CI improvement: Build the project not only on tags

### PR DESCRIPTION
**Pull request by mikheevm**
This PR:
- Build the project on every commit, not only on tags so we can identify commits breaking the build
- Dependency bump

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,4 +1,3 @@
-if: (branch = master) OR (branch = development) OR (tag IS present)
```
Comment (nicosampler) [inline .travis.yml:1]: perhaps you can add this condition, instead of removing it.
`type = pull_request`

Comment (mikheevm) [reply .travis.yml:1]: @denisgranha @giacomolicari wdyt?

Comment (mikheevm) [reply .travis.yml:1]: @nicosampler i'm not sure whether including all the possible cases is better than removing this condition at all (which will cover all the cases by default)

**Comment by mikheevm** [comment]
closed in favor of https://github.com/gnosis/safe-react-components/pull/19

**Pull request closed without merging**